equinix/resource_fabric_service_profile_acc_test.go

```diff
@@ -17,139 +17,140 @@ import (
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
 )
 
-func testAccFabricReadServiceProfileConfig(uuid string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profile" "test" {
-	uuid = "%s"
-	}
-`, uuid)
-}
+func TestAccFabricCreateServiceProfile_PFCR(t *testing.T) {
+	ports := GetFabricEnvPorts(t)
 
-func testAccFabricReadServiceProfilesListConfig(name string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profiles" "test" {
-		filter {
-			property = "/name"
-			operator = "="
-			values = ["%s"]
-		}
+	var portUuidDot1Q, portMetroCodeDot1Q string
+	var portUuidQinq, portMetroCodeQinq string
+	if len(ports) > 0 {
+		portUuidDot1Q = ports["pfcr"]["dot1q"][0].Uuid
+		portTypeDot1Q = ports["pfcr"]["dot1q"][0].type
```

Comment:
The variable is not declared before assignment, and again, `.type` is not a part of the v4.Port struct. Please update.